clients/src/main/java/org/apache/kafka/common/requests/LeaveGroupResponse.java

```diff
@@ -72,4 +73,15 @@ public static LeaveGroupResponse parse(ByteBuffer buffer, short versionId) {
     public boolean shouldClientThrottle(short version) {
         return version >= 2;
     }
+
+    @Override
```

Comment:
Is it necessary to add `equals` and `hashCode` in this fix?

Reply:
It should be useful for unit test purpose.